webcompat/static/css/development/components/markdown.css

```diff
@@ -21,9 +21,8 @@
   /* all element */
   .wc-Markdown * {
     font-size: var(--Markdown-fontSize) !important;
-    word-wrap: break-word;
-    white-space: normal;
```

Comment:
Altogether, it was not necessary to add this part. It was enough to target <code>, I wanted to clean up a bit the Markdown code. Maybe I should not have. In any case, I added the comment (http://localhost:5000/issues/667#issuecomment-306398355) At the origin of the previous PR, to see if there is not regression at the same time.